tests/observability/metrics/utils.py

```
@@ -655,53 +653,47 @@ def create_windows11_wsl2_vm(
+            firmware=base_spec.get("firmware"),
+            requirements=base_spec.get("requirements"),
+        ) as preference:
+            with VirtualMachineForTests(
+                os_flavor=OS_FLAVOR_WINDOWS,
+                name=vm_name,
+                namespace=namespace,
+                client=client,
+                vm_instance_type=VirtualMachineClusterInstancetype(client=client, name="u1.large"),
+                vm_preference=preference,
+                data_volume_template={"metadata": dv.res["metadata"], "spec": dv.res["spec"]},
+            ) as vm:
```

Ruff (0.16.0)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)